apps/hash-frontend/src/pages/[shortname]/shared/flow-visualizer/topbar.tsx

```diff
@@ -191,7 +191,10 @@ export const Topbar = ({
                 >
                   Run
                   {run.closedAt
-                    ? ` – ${format(new Date(run.closedAt), "yyyy-MM-dd h:mm a")}`
+                    ? ` – ${format(
+                        new Date(run.closedAt),
+                        "yyyy-MM-dd h:mm a",
+                      )}`
```

Comment:
I'm curious where these diffs are coming from, I don't see any changes to prettier in this PR - is it coming from your editor?

Reply:
I don't know, I tried to figure this out - linting is passing and formatting should be deterministic... I also didn't touch or open some of these files. There was a recent patch prettier update, dunno if that's done something.

I also get warnings on prettier lint locally now and AFAIR we aren't supposed to have warnings

Reply:
Will see if it starts happening in other PRs and look further if so